build.py

```diff
@@ -151,23 +151,23 @@ def build_variable_fonts(designspace, *steps):
 
     print(f"[{familyName}] Done: {file_path}")
 
-    print(f"[{familyName}] Compiling CFF2")
-    file_path_cff2 = (OUTPUT_DIR / file_stem).with_suffix(f".otf")
+    #print(f"[{familyName}] Compiling CFF2")
```

Comment:
oop

Reply:
:waves hand: These aren't the variable fonts you're looking for.